### PR TITLE
22

### PR DESCRIPTION
**Pull request by osaosula**
Menu Items reordered

### CODE REVIEW

File: src/MenuItem.jsx [hunk 1]
```diff
@@ -11,37 +11,37 @@ function MenuItem() {
...
             component={RouterLink}
-            to="/employees"
+            to="/admin"
             sx={{ textDecoration: "none" }}
           >
-            Employee Management
+            Admin
           </Link>
-        </ListItem>
+        </ListItem>{" "}
```
Comment (BorisLaboratory) [inline src/MenuItem.jsx:19]: Redundant comment

Comment (BorisLaboratory) [inline src/MenuItem.jsx:19]: Redundant comment

**Review by BorisLaboratory — CHANGES_REQUESTED** [review]

**Review by BorisLaboratory — CHANGES_REQUESTED** [review]
Some redundant comment, otherwise it looks good

File: src/MenuItem.jsx (hunk 1 above)
Comment (osaosula) [reply src/MenuItem.jsx]: It's the IDE that added it... lol 
Fixed!!

**Review by BorisLaboratory — APPROVED** [review]
Looks good to me

**Pull request merged**